src/scheduler/Makefile.am

```diff
@@ -136,5 +136,5 @@ dist_sysconf_DATA = \
 	pbs_sched_config
 
 if UNDOLR_ENABLED
-common_libs += @libundolr_lib@
+pbs_sched_LDADD += @libundolr_lib@
```

Comment:
You shouldn't need to check UNDOLR_ENABLED since libundolr_lib should be an empty string if it was not requested. In that case you can just add @libundolr_lib@ after ${common_libs} where pbs_sched_LDADD is defined. What you currently have will work, but you have to examine the whole file to see that it's only added for pbs_sched.